### ISSUE TITLE
QItem on IE11, Edge: With display:block takes up additional space below it

### ISSUE THREAD

**Issue by MartinManev** [quasarframework/quasar#5656]
**Describe the bug**
When I set `display: block;` on a QItem, then that item takes up additional space below it. This is only present on IE11 and Edge. If I inspect the element in said browsers I can see that there are the following rules applied:
```css
.q-item::after{
  min-height: 32px !important;
}
.row::after, .flex::after{
  content: "";
  display: block;
  font-size: 0px;
  min-height: inherit;
}
```
I am not seeing these rules in the other browsers.
I tracked the source of these rules to `https://cdn.jsdelivr.net/npm/quasar@1.4.4/dist/quasar.css`:
```css
@supports (-ms-ime-align: auto) {
  .row:after,
  .flex:after {
    content: '';
    display: block;
    font-size: 0;
    min-height: inherit;
  }
  .q-notification:after,
  .q-banner:after,
  .q-field__native:after,
  .q-field__control:after {
    min-height: 0 !important;
  }
  .q-item:after {
    min-height: 32px !important;
  }
  .q-list--dense > .q-item:after,
  .q-item--dense:after {
    min-height: 24px !important;
  }
  .q-btn__content,
  .q-time__content,
  .q-toolbar__title,
  .q-menu .q-item__section--main {
    flex-basis: auto;
  }
  .q-banner__content,
  .q-notification__message {
    flex-basis: 0 !important;
  }
  .q-menu > .q-banner > .q-banner__content,
  .q-dialog__inner > .q-banner > .q-banner__content {
    flex-basis: auto !important;
  }
  .q-tab__content {
    flex-basis: auto;
    min-width: 100%;
  }
  .column {
    min-width: 0%;
  }
```
While these rules do not interfere when `display: block;` is omitted and are probably there for another reason, they do create a problem here.

**Codepen/jsFiddle/Codesandbox (required)**
https://jsfiddle.net/3byah892/1/

**To Reproduce**
1. Open the jsFiddle example and see the problem on IE11 or Edge.
2. Play around with `display: block`

**Expected behavior**
Not take up additional space just like in the other browsers.

**Platform (please complete the following information):**
OS: Windows
Browsers: IE11, Edge

**Additional context**
My use case: I have a QItem with `display: block` nested inside a QList. I need `display: block` to make it take up the available width and play nicely with slide(left/right) transitions.
I might be able to find a workaround by using a different component/element or setting some other CSS rule, but this already works great in the other browsers and I want to avoid doing anything specific for IE11 and Edge.

**Comment by MartinManev** [comment]
**Another similar problem**

https://jsfiddle.net/19abgf0q/1/

I've set `display: block;` to make the outer QItem take all of the available width.
The inner QItem has `justify-content: space-between;` to make the buttons go to each side.
But the extra content acts as a third element and so the second button ends up near the middle.

This is, again, not present on other browsers. Just in IE11 and Edge.

**Comment by rstoenescu** [comment]
Hi,

Setting "display: block" essentially overrides the default style of QItem.
For your use-case, just set "width: 100%" or the helper css class "full-width".

All components in Quasar are flexbox-based.

**Comment by MartinManev** [comment]
Hi, @rstoenescu,

I found the same problem when using `QCardActions`:
https://jsfiddle.net/hf5rk7wa/2/

These problems are caused by the extra CSS rules that are applied on Edge and IE11.
Leaving `display:block` aside, I am using the components as intended.
Are you suggesting that I do not use QItem or QCardActions because of these issues?
Could you give some information as to why there are these special CSS rules in the first place?

Regards

**Comment by rstoenescu** [comment]
@pdanpdan can you take it over from here, pls? thx

**Comment by rstoenescu** [comment]
This fix will be available in "quasar" v1.5.0
